### PR TITLE
toc, how-to/deploy: add deploying TiDB using TiUP

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

- Add `/how-to/deploy/orchestrated/` about deploying TiDB using TiUP
- Add the new file to `TOC.md`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/2635, https://github.com/pingcap/docs-cn/pull/2651
- Other reference link(s):

### CODE REVIEW

File: how-to/deploy/orchestrated/tiup.md [hunk 1]
```diff
@@ -0,0 +1,1582 @@
+---
+title: Deploy a TiDB Cluster Using TiUP
+summary: Learn how to easily deploy a TiDB cluster using TiUP.
+category: how-to
+---
+
+# Deploy a TiDB Cluster Using TiUP
+
+[TiUP](https://github.com/pingcap-incubator/tiup-cluster) is a TiDB operation and maintenance tool written in Golang. TiUP cluster is a cluster management component provided by TiUP. By using TiUP cluster, you can easily perform daily database operations, including deploying, starting, stopping, destroying, scaling, and upgrading a TiDB cluster; managing TiDB cluster parameters; deploying TiDB Binlog; deploying TiFlash; etc.
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:9]: [TiUP](https://github.com/pingcap-incubator/tiup) is a TiDB operation and maintenance tool introduced in TiDB 4.0. [TiUP cluster](https://github.com/pingcap-incubator/tiup-cluster) is a cluster management component provided by TiUP.

Comment (yikeke) [reply how-to/deploy/orchestrated/tiup.md:9]: 这部分属于 https://github.com/pingcap/docs-cn/pull/2662/files 对齐内容哈，我还没提对齐的 PR 出来～晚点提一下～

Comment (lilin90) [reply how-to/deploy/orchestrated/tiup.md:9]: 👌

**Pull request merged**